examples/mssql-failover-replica/main.tf

```diff
@@ -24,7 +24,7 @@ locals {
 
 module "mssql1" {
   source  = "terraform-google-modules/sql-db/google//modules/mssql"
-  version = "~> 21.0"
+  version = "~> 21.0.0"
```

Comment:
@drewmercer-bl whats the reason for all these changes? Plz revert it back to what it was.

Reply:
Yesterday I was getting `make docker lint` errors where it could not find the module versions unless I added the third version digit. I just pulled `examples` from `master` and ran it again and the issue has resolved itself.

Reply:
@drewmercer-bl after your lint test plz set it back to 22.0 like it is. Test will not fail as we execute a command to replace all these lines before running lint test in our CI. For reference [this is the line](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/f59328e52ef835fd07d4634b30be300d92aa4820/build/int.cloudbuild.yaml#L17) which takes care of it.

Reply:
I've reset back to `22.0`.  Is `make docker lint` the correct test to run locally or should I have been running something else?

Reply:
Not sure why Git is showing the files as changed... I pulled the entire examples directory straight from `master`

Reply:
Also, is this version mismatch intentional? https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/examples/mssql-failover-replica/main.tf#L27-L59

Reply:
@drewmercer-bl u can ignore version errors. Just make sure other lint test pass.

Reply:
Fixed.